tests/unittests/Caffe2ImporterTest.cpp

```diff
@@ -1897,3 +1897,196 @@ TEST(caffe2, SparseLengthsSum8BitsRowwise) {
 
   EXPECT_TRUE(expected.isEqual(result, 0.02f));
 }
+
+/// Test loading SparseLengthsWeightedSumFused8BitRowwise. This is created as a
+/// RowwiseQuantizedSparseLengthsWeightedSumNode. The following inputs/outputs
+/// are used/expected for this test. Note that the DATA input is
+/// rowwise-quantized in the init_net proto.
+///    DATA  =   [[2.0, -0.5, 13]]
+///    WEIGHTS = [3, 1, 0, 0, 0, 0, 2, -0.5]
+///    INDICES = [1, 0, 2, 0, 1, 2, 2, 0]
+///    LENGTHS = [3, 0, 3, 2]
+///    OUTPUT =  [[0.5, 0, 0, 25]]
+TEST(caffe2, SparseLengthsWeightedSumFused8BitRowwise) {
+  ExecutionEngine EE{BackendKind::Interpreter};
```

Comment:
can you do some on CPU backend as well, since we do verification against expected values.

Reply:
Well the OperatorTest already runs on the CPU backend. Why do you think it would be beneficial to test loading both on the CPU and Interpreter here? We don't do any of our Caffe2ImporterTests on the CPU backend.

Reply:
this is a bit controversial, but C2Importer should test the import part not execution.
If it tests execution, why does that test only a single backend while there are impls in other backends as well.
I do not want to block on this anyway. Good to go.